### PR TITLE
auto upload sholuld be auto import

### PR DESCRIPTION
**Pull request by yaacov**
## 📝 Description

auto upload should be auto import

## 🎥 Demo
Before:
import-upload-before

After:
import-upload-after

### CODE REVIEW

File: src/utils/resources/template/utils/constants.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export enum BOOT_SOURCE {
 export const BOOT_SOURCE_LABELS = {
   [BOOT_SOURCE.PVC]: 'PVC',
   [BOOT_SOURCE.DATA_SOURCE]: 'PVC',
-  [BOOT_SOURCE.DATA_SOURCE_AUTO_UPLOAD]: 'PVC (auto upload)',
+  [BOOT_SOURCE.DATA_SOURCE_AUTO_UPLOAD]: 'PVC (auto import)',
```
Comment (metalice) [inline src/utils/resources/template/utils/constants.ts:99]: Consider changing key value AUTO_UPLOAD to AUTO_IMPORT

Comment (yaacov) [reply src/utils/resources/template/utils/constants.ts:99]: nice +1

Comment (yaacov) [reply src/utils/resources/template/utils/constants.ts]: @metalice updated, please re-review

**Pull request merged**